docs/data/date-pickers/custom-field/behavior-autocomplete/MaterialDatePicker.tsx

```diff
@@ -36,7 +36,7 @@ function AutocompleteField(props: AutocompleteFieldProps) {
     ...other
```

Comment:
- Add the validation error in the `onChange` context
- Don't check if `onChange` exists since it always does when used inside a picker